### ISSUE TITLE
Call to Action Media (left configuration)

### ISSUE THREAD

**Issue by kgarza-utsa** [utsa-asc/college-dls#368]
for any of the configurations where the image is on the left side, can we move the clipped corner to the bottom left?

https://utsa-asc.github.io/college-dls/components/detail/call-to-action-media--left.html
https://utsa-asc.github.io/college-dls/components/detail/call-to-action-media--2up-left.html
https://utsa-asc.github.io/college-dls/components/detail/call-to-action-media--3up-left.html
https://utsa-asc.github.io/college-dls/components/detail/call-to-action-media--blue-left.html
https://utsa-asc.github.io/college-dls/components/detail/call-to-action-media--orange-left.html

in one of the banner components, there is an image that uses this class to place the clip mass on the bottom left
`.clip-mask-left-bottom`

**Comment by kgarza-utsa** [comment]
can we create a series of classes that can be used on any images that expand on the existing `.clip-top-img`?? 

`.clip-top-img` (clip on top right) 
`.clip-bottom-img` (clip on bottom left)
`.clip-top-alt-img` (clip on top left)
`.clip-bottom-alt-img` (clip on bottom right)

**Comment by garza-utsa** [comment]
this sounds like two different issues, we can break out all the clip top classes into a separate issue.

for consistency, I see value in flipping the position of this clip to the bottom left for left facing media variants, but we can do it after the utility classes are created.

**Comment by garza-utsa** [comment]
depends on #370
